### ISSUE TITLE
Use the budget that is set in the MoneyMoney category

### ISSUE THREAD

**Issue by MorrisJobke** [Xiphe/budgetbudget#24]
On initial import it would be helpful to use the budgets from the MoneyMoney categories directly.

**Comment by Xiphe** [comment]
I agree. Sadly this data is not accessible via the [AppleScript API](https://moneymoney-app.com/applescript/). 
I already plan to add a feature request for MoneyMoney to add a category export API and including the existent budget data would make sense in such API.

That said. I don't see this as a priority right now.

**Comment by Xiphe** [comment]
Whats interesting to me: I personally have never used the budgeting feature of MM. and therefore have no concept of how this could be mapped into BB. 

I understand that the budgets have no time-demension and after realizing that I stopped thinking of them :)

Would you mind sharing how you use them?

**Comment by MorrisJobke** [comment]
> Would you mind sharing how you use them?

Sure:

It is basically an entry on the left side of the category in the sidebar. It shows the budget left (green if within the budget, yellow if towards the end of the budget and red if over the budget). And then there is also the currently spend money also shown, which is a bit confusing, but okay. So far the UI.

I use then categories and subcategories. Then some (most of my categories) have this budget set and I can easily spot where the budget is on track and where I overspend or will overspend if more comes in. Another useful thing there is to see in the overview at the top to have a line with the budget. So you see in which months you have been over the budget (and as you said the negative part here is that there is no time based budget, but only one per category). So I maintain the budget of the month in a spreadsheet and import the expenses there and export the new budgets from there.

But this line in the overview is really nice, because you can then switch from monthly to quarterly or yearly view and then have the accumulated budget also visible. This give you a hint if some months were totally of or are compensated by previous months.

So I don't use them extensively due to the lack of options, but they give a good hint.

**Comment by Xiphe** [comment]
Awesome! Thanks for sharing. I have not noticed the dotted line in the interface when I tried it.
This helps me a lot to understand how the feature is actually being used in MM. 👍

So MM is basically just assuming that you have the budgeted amount available in the given timeframe and there is no "total Budgeted" vs. "income". counter right?

**Comment by Xiphe** [comment]
Blocked by #39

**Comment by MorrisJobke** [comment]
> So MM is basically just assuming that you have the budgeted amount available in the given timeframe and there is no "total Budgeted" vs. "income". counter right?

Correct. It's just a "this budget is set - so it was there since ever and will there be in the future" also there is no carry-over from a previous month.
